src/http.ts

```diff
@@ -1250,7 +1250,7 @@ export class LemmyHttp {
     return this.#wrapper<GetFederatedInstances, GetFederatedInstancesResponse>(
       HttpType.Get,
       "/federated_instances",
-      form
+      form,
```

Comment:
If we weren't enforcing trailing commas before, we might as well override that rule in the prettierrc.

Reply:
I'd much rather use default prettier settings, rather than do overrides. Trailing commas got added in newer versions of prettier, so let's stick to that.